Xamarin.Forms.Platform.UAP/ListViewRenderer.cs

```diff
@@ -97,32 +97,55 @@ protected override void OnElementChanged(ElementChangedEventArgs<ListView> e)
 
 		bool IsObservableCollection(object source)
 		{
-			var type = source.GetType();
-			return type.IsGenericType &&
-				   type.GetGenericTypeDefinition() == typeof(ObservableCollection<>);
+			/* #fix=7167
```

Comment:
Please delete the comment. :)

Reply:
Got already deleted with the last commit. :)